runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectMetricsTest.java

```diff
@@ -82,15 +84,20 @@ public void testApplyLogicalQueryNoFilter() {
 
     MetricQueryResults results = metrics.queryMetrics(MetricsFilter.builder().build());
     assertThat(results.counters(), containsInAnyOrder(
-        metricResult("ns1", "name1", "step1", 5L, 0L),
-        metricResult("ns1", "name2", "step1", 12L, 0L),
-        metricResult("ns1", "name1", "step2", 7L, 0L)));
+        attemptedMetricsResult("ns1", "name1", "step1", 0L),
+        attemptedMetricsResult("ns1", "name2", "step1", 0L),
+        attemptedMetricsResult("ns1", "name1", "step2", 0L)));
+    assertThat(results.counters(), containsInAnyOrder(
+        committedMetricsResult("ns1", "name1", "step1", 5L),
+        committedMetricsResult("ns1", "name2", "step1", 12L),
+        committedMetricsResult("ns1", "name1", "step2", 7L)));
+    assertThat(results.distributions(), contains(
+        attemptedMetricsResult("ns1", "name1", "step1", DistributionResult.ZERO)));
     assertThat(results.distributions(), contains(
-        metricResult("ns1", "name1", "step1",
-            DistributionResult.create(12, 3, 3, 5),
-            DistributionResult.ZERO)));
+        committedMetricsResult("ns1", "name1", "step1", DistributionResult.create(12, 3, 3, 5))));
   }
 
+  @SuppressWarnings("unchecked")
   @Test
   public void testApplyPhysicalCountersQueryOneNamespace() {
```

Comment:
While we're here, let's change `testApplyLogicalQueryNoFilter` to `testApplyCommittedNoFilter` and `testApplyPhysicalCountersQueryOneNamespace` to `testApplyAttemptedCountersQueryOneNamespace` (eg., `s/Logical/Committed/` and `s/Physical/Attempted`)